.github/workflows/screenshots.yml

```diff
@@ -47,33 +67,125 @@ jobs:
               if: ${{ github.ref != 'main' }}
               run: |
                   MAIN_COMMIT_SHA=$(git show-ref main --hash)
-                  echo "::set-output name=before::https://gu-mobile-pr-images.s3-eu-west-1.amazonaws.com/$MAIN_COMMIT_SHA/screenshot.png"
-                  echo "::set-output name=after::https://gu-mobile-pr-images.s3-eu-west-1.amazonaws.com/${{ github.sha }}/screenshot.png"
-            - name: Search for previous screenshot comment
-              if: ${{ github.event_name == 'pull_request' }}
-              uses: peter-evans/find-comment@v1
-              id: fc
-              with:
-                  issue-number: ${{ github.event.pull_request.number }}
-                  body-includes: <!-- github-actions-screenshots -->
+                  echo "::set-output name=before::https://gu-mobile-pr-images.s3-eu-west-1.amazonaws.com/$MAIN_COMMIT_SHA/standard/screenshot.png"
+                  echo "::set-output name=after::https://gu-mobile-pr-images.s3-eu-west-1.amazonaws.com/${{ github.sha }}/standard/screenshot.png"
             - name: Create comment if necessary
-              if: ${{ github.event_name == 'pull_request' && steps.fc.outputs.comment-id == '' }}
+              if: ${{ github.event_name == 'pull_request' }}
               uses: peter-evans/create-or-update-comment@v1
               with:
```

Comment:
If we set `replace` as the `edit-mode` here, then I don't think we'll need the `delete_screenshots` job?

Reply:
(Obviously we'd need to reinstate the find comment step too)